package.json

```diff
@@ -7,14 +7,19 @@
     "build": "vitepress build docs"
   },
   "devDependencies": {
-    "vitepress": "1.0.0-alpha.45",
+    "@warp-ds/component-classes": "1.0.0-alpha.6",
```

Comment:
should we bump to 13?

Reply:
Fixed in https://github.com/warp-ds/tech-docs/pull/1/commits/6f23e0089ae8d0a9fc38968a2f387cdf2bfa806c